src/ol/style/fillstyle.js

```diff
@@ -26,6 +26,12 @@ ol.style.Fill = function(opt_options) {
 
   /**
    * @private
+   * @type {function(CanvasRenderingContext2D)}
+   */
+  this.renderer_ = options.renderer !== undefined ? options.renderer : null;
```

Comment:
This option needs to be added to `olx.style.FillOptions` in `externs/olx.js`, otherwise this will only work in debug mode. And the option should be documented there.